ext/nnstreamer/tensor_decoder/tensordec-boundingbox.c

```diff
@@ -262,6 +263,7 @@ loadImageLabels (bounding_boxes * data)
   labels = g_strsplit (contents, "\n", -1);
   data->total_labels = g_strv_length (labels);
   data->labels = g_new0 (char *, data->total_labels);
+  g_assert (data->labels != NULL);
```

Comment:
Let's make some "handle-able" error returns instead of assert so that we can let applications do something before killing itself.